src/mpid/ch4/netmod/ofi/util.c

```diff
@@ -230,10 +230,12 @@ static bool check_mpi_acc_valid(MPI_Datatype dtype, MPI_Op op)
     return valid_flag;
 }
 
-static void mpi_to_ofi(MPI_Datatype dt, enum fi_datatype *fi_dt, MPI_Op op, enum fi_op *fi_op)
+int MPIDI_mpi_to_ofi(MPI_Datatype dt, enum fi_datatype *fi_dt, MPI_Op op, enum fi_op *fi_op)
```

Comment:
Prefix should be `MPIDI_OFI_...` since this is limited to ofi netmod.

Reply:
Fixed